docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/01_ModelAdmin.md

```diff
@@ -114,25 +114,39 @@ class Category extends DataObject
 {
     public function canView($member = null) 
     {
-        return Permission::check('CMS_ACCESS_MyAdmin', 'any', $member);
+        return Permission::check('CMS_ACCESS_Company\Website\MyAdmin', 'any', $member);
     }
 
     public function canEdit($member = null) 
     {
-        return Permission::check('CMS_ACCESS_MyAdmin', 'any', $member);
+        return Permission::check('CMS_ACCESS_Company\Website\MyAdmin', 'any', $member);
     }
 
     public function canDelete($member = null) 
     {
-        return Permission::check('CMS_ACCESS_MyAdmin', 'any', $member);
+        return Permission::check('CMS_ACCESS_Company\Website\MyAdmin', 'any', $member);
     }
 
     public function canCreate($member = null) 
     {
-        return Permission::check('CMS_ACCESS_MyAdmin', 'any', $member);
+        return Permission::check('CMS_ACCESS_Company\Website\MyAdmin', 'any', $member);
     }
 }
 ```
+## Custom ModelAdmin CSS Menu Icons using built in Icon Font
```

Comment:
Would you mind removing the capitalisations here aside from “Custom ModelAdmin”?

Reply:
Yes, done, thanks